DesktopUI2/DesktopUI2/ViewModels/AccountViewModel.cs

```diff
@@ -171,7 +171,7 @@ private void DownloadComplete(object sender, DownloadDataCompletedEventArgs e)
       }
       catch (Exception ex)
       {
-        Log.Debug(ex, "Swallowing exception {exceptionMessage}",ex.Message);
+        Log.Warning(ex, "Swallowing exception {exceptionMessage}",ex.Message);
```

Comment:
Would it make sense to add more content here, like "Swallowing exception in avatar download"?
Sometimes the full trace is lost, and it's tricky to understand where an exception was thrown...

Reply:
yeah, you are right. Hopefully with the Exception rework efforts we've made the stack traces more consistent, but its prob better to add extra context.

I propose making it like this:
```
Log.Warning(ex, "Swallowing exception in {methodName}: {exceptionMessage}", nameof(method), ex.Message);
```

Reply:
👍 I'll make these changes

Reply:
I've added the method name into all "Swallowing" exception templated logs, as per your proposal.
However, perhaps there is some value in enriching all our logs like this.

https://stackoverflow.com/questions/29470863/serilog-output-enrich-all-messages-with-methodname-from-which-log-entry-was-ca

Looks like it may be possible either through a custom ILogger extension method (with `CallerMemberNameAttribute`), or through `AutofacSerilogIntegration` to enrich the LoggerConfiguration